kafka-ui-react-app/src/components/Topics/Topic/Details/Messages/Filters/Filters.tsx

```diff
@@ -165,6 +171,20 @@ const Filters: React.FC<FiltersProps> = ({
     };
   }, [attempt, query, queryType, seekDirection, activeFilter]);
 
+  const handleClearAllFilters = () => {
+    setCurrentSeekType(SeekType.OFFSET);
+    setQuery('');
+    getSelectedPartitionsFromSeekToParam(searchParams, partitions);
+    setSelectedPartitions(
```

Comment:
should we reset the seekDirection as well ?

Reply:
done